_posts/2024-09-24-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2024.39
+author: Priya
+date: 2024-09-24
+category: Jekyll
+layout: post
+---
+### Enhancement:
+Added a sample flow to the onboarding process, enabling users to visualize data transformation through an example workflow.
```

Comment:
Priya, made some changes to this message - how does this sound?
"Added a sample flow to the onboarding process, to help users understand setting up a data transformation workflow in Mammoth through sample data"